src/Options/ParseOptions.cpp

```diff
@@ -0,0 +1,64 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <exception>
+#include <string>
+#include <vector>
+#include <yaml-cpp/yaml.h>
+
+#include "Informer/InfoFromBuild.hpp"
+#include "Utilities/FileSystem.hpp"
+
+namespace Options::detail {
+namespace {
+void check_metadata(const YAML::Node& metadata) {
+  // Validate executable name
+  if (const auto& exec_name = metadata["Executable"]) {
+    if (file_system::get_file_name(exec_name.as<std::string>()) !=
```

Comment:
When would this be a path?

Reply:
We may use the "Executable" field to tell tools which executable to run, in which case we sometimes want to specify an absolute path.

Reply:
OK.  I can imagine that being useful for batch runs or something.